data/json/monsterdrops/zombie.json

```
@@ -294,5 +294,11 @@
       { "group": "fishing_items_trash", "count": [ 2, 4 ] },
       { "item": "withered", "count": [ 10, 20 ] }
     ]
+  },
+  {
+    "type": "item_group",
+    "id": "mon_dog_death_drops",
+    "subtype": "collection",
+    "entries": [ { "group": "dog_clothes", "damage": [ 1, 4 ], "prob": 20 } ]
```

That's fairly high for what I assume would be every single dog in the game, which will be substantially more common than z-9:
```suggestion
    "entries": [ { "group": "dog_clothes", "damage": [ 1, 4 ], "prob": 5 } ]
```